app/Console/Commands/CreateDefaultPrinter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Printer;
+use Illuminate\Console\Command;
+
+class CreateDefaultPrinter extends Command
```

Comment:
This is basically an initialization command, so that inserting a printer into the db by hand is not needed. I can add customization if you think it's necessary.

Reply:
But when do you call it? Can't you just create it in a migrations then?